embedchain/embedchain.py

```diff
@@ -50,6 +50,14 @@ def __init__(self, config: BaseAppConfig):
         thread_telemetry = threading.Thread(target=self._send_telemetry_event, args=("init",))
         thread_telemetry.start()
 
+    @property
+    def collect_metrics(self):
+        return self.config.collect_metrics
+
+    @collect_metrics.setter
+    def collect_metrics(self, value):
+        self.config.collect_metrics = value
```

Comment:
I think it will be nice to have a type validation, given config.collect_metrics is a boolean type. Something like

```
    @collect_metrics.setter
    def collect_metrics(self, value):
        if not isinstance(value, bool):
            raise ValueError(f'Boolean value expected, got {type(value)}.')
        self.config.collect_metrics = value
```

Reply:
Yes, its important to have it. Completely forgot about it. Will make the change